src/debug.c

```diff
@@ -1884,7 +1886,7 @@ static void writeStacktraces(int fd, int uplevel) {
 
     size_t skipped = 0;
 
-    char buff[MAX_BUFF_LENGTH];
+    char buff[PATH_MAX];
```

Comment:
Just a note, using `PATH_MAX` for a buffer implies that you are going to write a file path into it. If you are not using the buffer for this purpose, I would avoid doing that.

Reply:
please suggest an alternative macro

Reply:
If I'm building a path string, I use PATH_MAX but otherwise, if I'm building log lines as in here, I generally just use big enough one like `char buf[1024]` (ofcourse you can define something like MAX_BUFF_LENGTH if you want) 

Also, for snprintf, I generally pass local buffer size with sizeof() (whether buf size is a macro or not). The reasoning is that if I change buffer size later, no need to change in every snprintf invocation in the function. 

```c
char buf[1024];

snprintf(buf, sizeof(buf), "error occured: %d", errno); 
```

These are generally quite personal. So, please ignore if you disagree :)